Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -97,3 +97,4 @@
 --rules preferForLoop
 --rules conditionalAssignment
 --rules wrapMultilineConditionalAssignment
+--rules preferInferredTypes
```

Comment:
Will pull in a SwiftFormat build with this new rule after https://github.com/nicklockwood/SwiftFormat/pull/1640 is merged.